agent_api/src/main/java/dev/aikido/agent_api/vulnerabilities/SkipVulnerabilityScanDecider.java

```diff
@@ -22,7 +22,10 @@ public static boolean shouldSkipVulnerabilityScan(ContextObject context) {
             }
             List<Endpoint> matchedEndpoints = matchEndpoints(context.getRouteMetadata(), threadCache.getEndpoints());
             if (matchedEndpoints != null && !matchedEndpoints.isEmpty()) {
```

Comment:
Maybe the empty check isn't needed? `anyMatch(...)` returns `false` if empty?

Reply:
true